### PR TITLE
feat: fix feature tests for LP resizing

### PR DESCRIPTION
**Pull request by Jiajia-Cui**
link to [PR: LP order sizing #7016](https://github.com/vegaprotocol/vega/pull/7016)

### CODE REVIEW

File: core/integration/features/position_tracking/verified-positions-resolution-5-lognormal.feature [hunk 1]
```diff
@@ -57,95 +57,42 @@ Feature: Position resolution case 5 lognormal risk model
...
-      #| party  | asset | market id | margin        | general | bond |
-      #| lpprov | USD   | ETH/DEC19 | 1000000000000 | 0       | 0    |
+      | party  | asset | market id | margin  | general      | bond  |
+      | lpprov | USD   | ETH/DEC19 | 6907938 | 999993018136 | 90000 |
 
     Then the parties should have the following account balances:
-      | party            | asset | market id | margin  | general |
-      | designatedLooser | USD   | ETH/DEC19 | 21600   | 0       |
+      | party            | asset | market id | margin | general |
+      | designatedLooser | USD   | ETH/DEC19 | 0      | 0       |
```
Comment (davidsiska-vega) [inline core/integration/features/position_tracking/verified-positions-resolution-5-lognormal.feature:82]: Looks like the party got closed out where previously it was still alive - is this intended?

File: core/integration/features/position_tracking/verified-positions-resolution-5-lognormal_pdp.feature [hunk 2]
```diff
@@ -40,95 +40,54 @@ Feature: Position resolution case 5 lognormal risk model
...
+    # insurance pool generation - trade
     When the parties place the following orders with ticks:
       | party            | market id | side | volume | price | resulting trades | type       | tif     | reference |
       | designatedLooser | ETH/DEC19 | buy  | 29000  | 150   | 1                | TYPE_LIMIT | TIF_GTC | ref-1     |
 
     Then the parties should have the following account balances:
-      | party            | asset | market id | margin  | general |
-      | designatedLooser | USD   | ETH/DEC19 | 21600   | 0       |
+      | party            | asset | market id | margin | general |
+      | designatedLooser | USD   | ETH/DEC19 | 0      | 0       |
```
Comment (davidsiska-vega) [inline core/integration/features/position_tracking/verified-positions-resolution-5-lognormal_pdp.feature:83]: Same question as above.

File: core/integration/features/price_monitoring/3362-failing-system-tests.feature [hunk 3]
```diff
@@ -56,36 +56,28 @@ Feature: Replicate failing system tests after changes to price monitoring (not t
...
-      | party2 | ETH/DEC20 | sell | 322    | 0     | 0                | TYPE_MARKET | TIF_FOK | OrderError: non-persistent order trades out of price bounds |
+      | party  | market id | side | volume | price  | resulting trades | type       | tif     |
+      | party2 | ETH/DEC20 | sell | 322    | 106000 | 0                | TYPE_LIMIT | TIF_GTC |
 
     Then the market data for the market "ETH/DEC20" should be:
       | mark price | last traded price | trading mode            | horizon | min bound | max bound |
       | 100000     | 100150            | TRADING_MODE_CONTINUOUS | 5       | 99845     | 100156    |
       | 100000     | 100150            | TRADING_MODE_CONTINUOUS | 10      | 99711     | 100290    |
 
-    ## Now place the order for the same volume again, but set price to 100,000 -> the buy at 95,000 doesn't uncross
```
Comment (davidsiska-vega) [inline core/integration/features/price_monitoring/3362-failing-system-tests.feature:81]: I think removing these means that the test is no longer testing what it was.

File: core/integration/features/settlement/mark_to_market_interval_positions.feature [hunk 4]
```diff
@@ -57,107 +57,60 @@ Feature: Check position tracking matches expected behaviour with MTM intervals.
...
     Then the parties should have the following margin levels:
-      | party           | market id | maintenance | search  | initial  | release |
-      | designatedLoser | ETH/DEC19 | 14654       | 17584   | 21981    | 29308   |
+      | party           | market id | maintenance | search | initial | release |
+      | designatedLoser | ETH/DEC19 | 14654       | 17584  | 21981   | 29308   |
 
     # Add another 4 blocks, and we will have crossed over the threshold, and we will MTM
     When the network moves ahead "4" blocks
     Then the parties should have the following margin levels:
-      | party           | market id | maintenance | search  | initial  | release |
```
Comment (davidsiska-vega) [inline core/integration/features/settlement/mark_to_market_interval_positions.feature:108]: Why would we be getting rid of all this?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
Looks all good - in the sense that the numerical values changed looks consistent with what the change in code is - apart from the comments I made.

File: core/integration/features/position_tracking/verified-positions-resolution-5-lognormal.feature (hunk 1 above)
Comment (Jiajia-Cui) [reply core/integration/features/position_tracking/verified-positions-resolution-5-lognormal.feature]: LP bond has been reduced, hence the party did not get close-out, and then close-out happens when the book has enough volume

File: core/integration/features/price_monitoring/3362-failing-system-tests.feature (hunk 3 above)
Comment (Jiajia-Cui) [reply core/integration/features/price_monitoring/3362-failing-system-tests.feature:81]: LP bond has been reduced, hence the party did not get close-out, and then close-out happens when the book has enough volume

**Pull request merged**